app.js

```diff
@@ -12,9 +16,12 @@ const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short';
 app.use(logger(formatsLogger));
 app.use(cors());
 app.use(express.json());
+app.use(morgan('dev'));
 
 app.use('/api/contacts', contactsRouter);
 app.use('/api/users', usersRouter);
+app.use('/avatars', avatarRouter, express.static(AVATAR_DIR));
+app.use('/users', usersAvatarRouter);
```

Comment:
W zadaniu niby jest PATCH /users/avatars ale zakładam że to pomija prefix. Raczej każde zapytanie na backend będziemy chcieli poprzedzać prefixem 'api'.